server/app/controllers/api/v1/experiences_controller.rb

```diff
@@ -0,0 +1,58 @@
+require 'uri'
+class Api::V1::ExperiencesController < ApplicationController
+  before_action :set_api_user
+  before_action :set_api_user_experience, only: [:show, :update, :destroy]
+
+  #GET /api_users/:api_user_id/experiences
+  def index
+    @experiences = @api_user.experiences.includes(:programminglanguage).includes(:meetinginterval)  if params[:api_user_id]
```

Comment:
According to the docs, you may specify multiple arguments to includes. 

https://apidock.com/rails/ActiveRecord/QueryMethods/includes

Reply:
This has been changed